src/actions/actions.js

```diff
@@ -17,6 +17,16 @@ export const getOneMonster = (url) => dispatch => {
         type: FETCH_ONE_MONSTER,
         payload: monster
     }))
+  .catch(error => console.log(error))
+}
+
+export const getMonstersByCR = (CR) => dispatch => {
+  fetch(`https://www.dnd5eapi.co/api/monsters?challenge_rating=${CR}`)
+      .then(response => response.json())
+       .then(CRMonsters => dispatch({
+        type: GET_MONSTERS_BY_CR,
+        payload: CRMonsters.results
+    }))
     .catch(error => console.log(error))
```

Comment:
Great job. We should make sure we are handling our errors more appropriately.

Reply:
Do we have a ticket for this?